en/firefox/developer.ftl

```diff
@@ -86,9 +86,21 @@ firefox-developer-view-the-release = View the <a { $attrs }>release notes</a> (E
 firefox-developer-developer-tools-user = Developer Tools User Docs
 firefox-developer-mdn-web-docs = { -brand-name-mdn-web-docs }
 firefox-developer-resources-for-developers = Resources for Developers, by Developers
+firefox-developer-mdn-playground = Playground
+firefox-developer-mdn-write-test-and-share = Write, test and share your code. Your playground to learn and share your amazing work with the world.
```

Comment:
The second sentence grammatically is incorrect.

Reply:
Why? It's a playground to learn and share work.

Reply:
My bad on this. I read it as "you" instead of "your".